### DISCUSSION TITLE
ResStock 2024.1 Release; Package Costs Information

### DISCUSSION THREAD

**Discussion by nroe413** [NatLabRockies/resstock#1216] [General]
Hello! 

I have recently gotten to check out the 2024.1 release and it is awesome! I feel so lucky that NREL does ResStock because it is so interesting and valuable! I was reading through the 2024.1 documentation (https://www.nrel.gov/docs/fy24osti/88109.pdf). 
On my pdf page 9 I see the following: 

_"Additional non-energy outputs are also possible from within ResStock if the appropriate inputs
are provided. Key among these are emissions calculations, utility bill calculations, and measure
package costs."_

Does ResStock track measure package "cost" for simulated households? I checked 1 or 2 metadata and annual results files but couldn't find a column tracking "package costs". Apologies if I am missing something and if these package costs are tracked could you please direct me! 

Cheers!

**Comment by whiphi92** [comment]
Hi @nroe413, thanks for getting in touch.

ResStock has the ability to track measure packages costs. The intention of the citation quote you provided is to inform the readers that ResStock has the capability to calculate measure package costs with provided inputs. For the 2024.1 release, measure package costs are not available - which is why you do not see any columns providing the costs.

-Phil

**Reply by nroe413** [reply]
Thanks, Phil! 

Would "provided inputs" be something that I come up with (from a non-resstock source) and then apply as multipliers based on the available data? Can you quickly explain how I could I apply these costs?

**Reply by whiphi92** [reply]
In metadata annual results, you should see some columns noted as "costs" along with, for example, "conditioned square feet" (for lighting, perhaps) or "sizing_heating_primary" (for HVAC costs). These are all values for a particular model you can use as multipliers with your own cost data.

**Comment by nroe413** [comment]
Okay I think I understand the physical parameters side of things. For
example, for a given heating type if I had my own cost/ft^2 for a given
space heating scheme with no backup I could check the value in the
"out.params.size_heating_system_primary_k_btu_h" column and then use my own
$/ft^2 cost multiplier to approximate the cost of that system.







 I guess I also have the question of these columns about "bills" (when is
ctrl+F searched for cost I just got the emissions levels/year under various
RE cost scenarios). The bills columns seem to reflect the annual cost of
some fuel resource ex: propane after the upgrade (if applicable) is applied.

 [image: image.png]

I am wondering:
1. Is this cost nominal dollars or real dollars, and if real, what year?
2. Can I "back out" the cost of the system based on say
"out.bills.propane.usd" based on the system efficiency, size, etc. since I
have a $/year (real or nominal) spent on propane and does ResStock work
through any examples of that type of process taking the bill cost and
working backward to calculate the heating system cost multiplier? Basically
I am trying to know if ResStock has its own cost information for equipment
and fuels that could produce a column like "out.bills.propane.usd"? I am
wondering if those data are published or are private?

Sorry for the massive email.

**Reply by whiphi92** [reply]
I think there might be some confusion. If you had heat pump cost structure data that is something like 10 dollars + 1 dollar per size in kbtuh (a hypothetical example), you could use our output for size_heating_system_primary_k_btu_h to calculate the cost using your own cost structure for each dwelling model.

1. See our documentation about calculating bill costs in the technical report.
2. There will not be a way to back out the cost of a system based on bills outputs, but you could back out the simple bill calculation structure we use. I think using the technical documentation on this is an easier exercise, though.

**Comment by nroe413** [comment]
Very helpful. Thank you for your patience and your explanation. I will
follow up with other confusion.

-Nate
